server/service/device_client.go

```diff
@@ -35,6 +38,26 @@ func (dc *DeviceClient) request(verb string, path string, query string, response
 	return dc.parseResponse(verb, path, response, responseDest)
 }
 
+func (dc *DeviceClient) SetToken(token string) {
+	dc.mu.Lock()
+	defer dc.mu.Unlock()
+	dc.token = token
+}
+
+func (dc *DeviceClient) GetToken() string {
```

Comment:
This is exported but never used outside `DeviceClient`

Reply:
great catch, thanks.